addons/plotting/initKeybinds.inc.sqf

```diff
@@ -0,0 +1,61 @@
+private _category = [ELSTRING(main,DisplayName), LSTRING(DisplayName)];
+
+[
+    _category,
+    QGVAR(toggleDistanceFormat),
+    [LSTRING(ToggleDistanceFormat), LSTRING(ToggleDistanceFormat_Description)],
+    {
+        if (!isNull curatorCamera && {!GETMVAR(RscDisplayCurator_search,false)}) then {
+            private _numFormatters = count GVAR(distanceFormatters);
+            GVAR(currentDistanceFormatter) = (GVAR(currentDistanceFormatter) + 1) % _numFormatters;
+
+            true
+        };
+    },
+    {},
+    [DIK_R, [false, false, false]] // Default: R
```

Comment:
These keybinds conflict with the same base game Zeus controls. I think a default behind some modifiers since this doesn't need to be so accessible would be good.

Reply:
Are these better? 
- <kbd>Ctrl</kbd>+<kbd>R</kbd>: Switch Distance
- <kbd>Shift</kbd>+<kbd>R</kbd>: Switch Speed
- <kbd>Ctrl</kbd>+<kbd>T</kbd>: Switch Azimuth